api/src/main/java/org/openmrs/module/eptsreports/reporting/utils/EptsCommonUtils.java

```diff
@@ -0,0 +1,115 @@
+import org.openmrs.module.reporting.data.patient.definition.PatientDataDefinition;
+import org.openmrs.module.reporting.data.patient.definition.PatientIdDataDefinition;
+import org.openmrs.module.reporting.data.patient.definition.PatientIdentifierDataDefinition;
+import org.openmrs.module.reporting.data.patient.definition.PersonToPatientDataDefinition;
+import org.openmrs.module.reporting.data.patient.definition.ProgramEnrollmentsForPatientDataDefinition;
+import org.openmrs.module.reporting.data.person.definition.BirthdateDataDefinition;
+import org.openmrs.module.reporting.data.person.definition.GenderDataDefinition;
+import org.openmrs.module.reporting.data.person.definition.PersonDataDefinition;
+import org.openmrs.module.reporting.data.person.definition.PreferredNameDataDefinition;
+import org.openmrs.module.reporting.dataset.definition.PatientDataSetDefinition;
+import org.openmrs.module.reporting.evaluation.parameter.Parameter;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+@Component
+public class EptsCommonUtils {
+
+  @Autowired private HivMetadata hivMetadata;
+  /**
+   * Adds the standard patient list columns
+   *
+   * @param dsd the data set definition
+   */
+  public void addStandardColumns(PatientDataSetDefinition dsd) {
+    // identifier
+    PatientIdentifierType artNo = hivMetadata.getNidServiceTarvIdentifierType();
+
+    DataConverter identifierFormatter = new ObjectFormatter("{identifier}");
+    DataDefinition identifierDef =
+        new ConvertedPatientDataDefinition(
+            "identifier",
+            new PatientIdentifierDataDefinition(artNo.getName(), artNo),
+            identifierFormatter);
+
+    dsd.addColumn("Patient Id", new PatientIdDataDefinition(), "");
+    dsd.addColumn("Patient NID", identifierDef, "");
+    dsd.addColumn("Patient Name", new PreferredNameDataDefinition(), (String) null);
+    dsd.addColumn(
+        "Patient Date of Birth",
+        convert(getBirthdateConverted(), new BirthdateConverter("dd-MM-yyyy")),
+        "");
+    dsd.addColumn(
+        "Estimated",
+        convert(getBirthdateConverted(), new PropertyConverter(Birthdate.class, "estimated")),
+        "");
+    dsd.addColumn("Sex", new PersonToPatientDataDefinition(new GenderDataDefinition()), "");
+    dsd.addColumn(
+        "First Entry Date",
+        getPatientDemographics("First Entry Date"),
+        "",
+        new CalculationResultDataConverter("F"));
+    dsd.addColumn(
+        "Last Updated",
+        getPatientDemographics("Last Updated"),
+        "",
+        new CalculationResultDataConverter("L"));
+  }
+
+  public DataDefinition getPatientDemographics(String name) {
+    return new CalculationDataDefinition(
+        name, Context.getRegisteredComponents(PatientDemographicsCalculation.class).get(0));
+  }
+
+  public DataDefinition getPatientProgramEnrollment(Program program, TimeQualifier timeQualifier) {
```

Comment:
These are not utility functions, better we create a new folder under `library` for data definitions, similar to how we have cohorts

Reply:
Moved those to a different file and package